process/smartContract/scQueryService.go

```diff
@@ -134,7 +134,7 @@ func (service *SCQueryService) ComputeScCallGasLimit(tx *transaction.Transaction
 		return 0, err
 	}
 
-	gasConsumed := service.economicsFee.MaxGasLimitPerBlock() - vmOutput.GasRemaining
+	gasConsumed := service.economicsFee.MaxGasLimitPerBlock(0) - vmOutput.GasRemaining
```

Comment:
It is ok to be harded here to shard 0 ?

Reply:
it is a query service - it does not matter.